framework/include/aifw/aifw_log.h

```diff
@@ -17,46 +17,45 @@
  ****************************************************************************/
 
 /**
- * @file lw_aifw/lw_aifw_log.h.h
- * @brief This file defines macros used to add debug logs in Light Weight AI Framework and it's applications.
+ * @file aifw/aifw_log.h
+ * @brief This file defines macros used to add debug logs in AI Framework and it's applications.
  *
  * Log levels are Error, Information and Verbose. Error is by default enabled.
  */
 
-#ifndef __LW_AIFW_LOGS_H__
-#define __LW_AIFW_LOGS_H__
+#ifndef __AIFW_LOGS_H__
+#define __AIFW_LOGS_H__
```

Comment:
When do you use `#pragma` or `#ifndef`?

Reply:
- #pragma once is not part of the standard and may not work on all operating systems or compilers. However, all major compilers support it on all major platforms.
- #ifndef is the standard compliant way method of including a file only once on most OS.
- #pragma once reduces possibilities for bugs because it is a lot easier to do refactoring.

Reply:
@aadotverma Sorry but this is not what I asked. You use both in new files. Some files use `#pragma` and others use `#ifndef`. As you said, both has adv and disadv so that we should have some policy. In your module, what will you use? just use both?

Reply:
#ifndef include guards replaced with #pragma once